src/expression-modifiers/normalization/index.js

```diff
@@ -0,0 +1,114 @@
+import util from '../../utils/util';
+import helper from '../helper';
+import propertyPanelDef from './properties';
+
+const DEFAULT_OPTIONS = {
+  type: 'normalization',
+  disabled: false,
+  primaryDimension: 0,
+  relativeNumbers: 0,
+  crossAllDimensions: false,
+  showExcludedValues: true,
+  fullNormalization: false,
+  outputExpression: '',
+};
+
+const maxNumDimensionsSupported = 2;
+
+function getAggrOneDim(expression, dim1) {
+  return `Aggr(${expression}, ${dim1})`;
+}
+
+function getAggrDisregadSelec(expression, dim1) {
```

Comment:
spelling error here - should be `Disregard`, preferrably call it `getAggrDisregardSelect` instead..